packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -111,7 +111,22 @@ const TEMPLATE_SOURCE_NPM = 'npm'
 const TEMPLATE_SOURCE_BUNDLE = 'bundle'
 const DEFAULT_TEMPLATE_VERSION = 'latest'
 
-const askApplicationExtensibiltyQuestions = (appExtensions) => {
+const APP_EXTENSIBILITY_TYPESCRIPT_MINIMAL_DEV_DIR = 'dev'
```

Comment:
It took me about 5 seconds just to read this variable name, and I didn't know what it did after all that :P .. Can we think of something else for this? 

I'm not sure we have to include the typescript minimal name in here as the template that is generated for that folder could potentially be variable. E.g. if we get feedback on people wanting a `javascript-minimal` project.

Reply:
I updated the naming to `APP_EXTENSIBILITY_DEV_PROJECT_DIR`. 
Other naming ideas: `DEV_DIR` or `DEV_PROJECT_DIR` or `LOCAL_DEV_DIR` or `LOCAL_DEV_PROJECT_DIR`.